app/models/study_json_record/processor_v2.rb

```diff
@@ -88,6 +88,65 @@ def brief_summary_data
   end
 
   def design_data
+    return unless protocol_section
+    nct_id = protocol_section.dig('identificationModule', 'nctId')
+    puts "NCT ID: #{nct_id}"

```

Comment:
Does this PUTS statement belong there, or was it only for testing?

Reply:
Oh thought I removed that thanks @Javier-Jimenez-18. I'll fix it now